services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unit_template_first_row.jsx

```diff
@@ -43,32 +48,57 @@ export default class UnitTemplateFirstRow extends React.Component {
     )
   }
 
+  renderActivityPackImage(data) {
+    if(!data) { return }
+
+    const { image_link, type } = data
+    const { unit_template_category } = data
```

Comment:
Maybe this is more readable, but do we need two different decompositions of `data`?

Reply:
no that was by accident, good catch